worker/file_util.py

```diff
@@ -272,6 +199,20 @@ def ensure_ends_with_newline(lines, remove_line_without_newline=False):
     
     return ''.join(lines)
 
+def get_path_size(path, ignore_paths=[]):
```

Comment:
ignore_names might be better because these are names of directories inside path, not absolute paths (also to be consistent with other parts of the code)